core/src/scss/utilities/variables/components/_nav.scss

```diff
@@ -0,0 +1,168 @@
+@charset "UTF-8";
+
+//
+// Nav Color Maps
+//
+// Color maps for the Main Nav and Secondary Nav Components.
+// "h-" prefix in the color names indicate horizontal orientation, which currently only applies to the top level items
+// of the desktop version of Main Nav component. Colors without the "h-" applies to the Secondary Nav (all items),
+// mobile version of the Main Nav, and the drop down menus of the desktop version of Main Nav.
+// Active color is also used to indicate "current" page.
+//
+// Styleguide: Variables.Components.nav-colors
+//
+
+//
+// $nav-colors
+//
+// Color maps for the default Main Nav and Secondary Nav Components.
+//
+// Markup: $nav-colors: (
+//    bg:                             $color-black,
+//    divider:                        $color-cool-grey,
+//    link:                           $color-white,
+//    link--hover:                    $color-white,
+//    link--active:                   $color-white,
+//    link--expanded:                 $color-white,
+//    border--hover:                  $color-white,
+//    border--active:                 $color-vivid-red,
+//    border--expanded:               transparent,
+//    border--active-expanded:        $color-white,
+//    h-link:                         $color-black,
+//    h-link--hover:                  $color-bright-red,
+//    h-link--active:                 $color-black,
+//    h-link--active-hover:           $color-bright-red,
+//    h-link--expanded:               $color-black,
+//    h-border--hover:                $color-bright-red,
+//    h-border--active:               $color-black,
+//    h-border--active-hover:         $color-bright-red,
+//    h-border--expanded:             $color-black,
+//) !default;
+//
+// Styleguide: Variables.Components.nav-colors.default
+//
+
+$nav-colors: (
```

Comment:
Please be aware that these variables, and the rest of the nav, are being refactored currently in: https://github.com/SU-SWS/decanter/pull/443/files#diff-f94ee2cd63a360f0412ca184c56b3af9

Continue with what you are doing here but they will likely change in v6.